### PR TITLE
(chore) O3-2934: Update urls in the README and PR template

### PR DESCRIPTION
**Pull request by kdaud**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

https://openmrs.atlassian.net/browse/O3-2934

## Screenshots

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ It is responsible for rendering the UI for the landing page after the user logs
 
 ## Contributing / Development
 
-[Setup local development environment for OpenMRS SPA](https://wiki.openmrs.org/display/projects/Setup+local+development+environment+for+OpenMRS+SPA)
+[Setup local development environment for OpenMRS SPA](https://wiki.openmrs.org/display/projects/Setup+local+development+environment+for+OpenMRS+SPA )
```
Comment (kdaud) [inline README.md:18]: @denniskigen could you be aware of the correct link to the local development set up instruction? The existing one link is pointing to a non public content.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
-:wave:	*New to our project? Be sure to review the [OpenMRS 3 Frontend Developer Documentation](https://openmrs.github.io/openmrs-esm-core/#/). You may find the [Map of the Project](https://openmrs.github.io/openmrs-esm-core/#/main/map) especially helpful.* :teacher:	
+:wave:	*New to our project? Be sure to review the [OpenMRS 3 Frontend Developer Documentation](https://openmrs.github.io/openmrs-esm-core/#/ ). You may find the [Map of the Project](https://openmrs.github.io/openmrs-esm-core/#/main/map) especially helpful.* :teacher:
```
Comment (kdaud) [inline README.md:1]: This link as well is pointing a wrong site.

**Review by kdaud — COMMENTED** [review]
@denniskigen I've done a self review. Could you be in position to address the reviews with a commit? Thanks.

File: README.md (hunk 2 above)
Comment (denniskigen) [reply README.md:1]: Use this as a guide https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/README.md

File: README.md (hunk 1 above)
Comment (denniskigen) [reply README.md:18]: https://o3-docs.openmrs.org/docs/frontend-modules/development

**Review by denniskigen — APPROVED** [review]
Thanks, @kdaud!

**Pull request merged**